// vanillajs/dest/es/index.js
var IntlJS = Intl;

var PromiseJS = Promise;

var ProxyJS = Proxy;

var ReflectJS = Reflect;

var WebAssemblyJS = WebAssembly;

var InfinityJS = Infinity;

var NaNJS = NaN;

var undefinedJS = undefined;

var decodeURIJS = decodeURI;

var decodeURIComponentJS = decodeURIComponent;

var encodeURIJS = encodeURI;

var encodeURIComponentJS = encodeURIComponent;

// eslint-disable-next-line no-restricted-globals
var isFiniteJS = isFinite;

// eslint-disable-next-line no-restricted-globals
var isNaNJS = isNaN;

var parseFloatJS = parseFloat;

var parseIntJS = parseInt;

var BooleanJS = Boolean;

var FunctionJS = Function;

var ObjectJS = Object;

var SymbolJS = Symbol;

var ErrorJS = Error;

var EvalErrorJS = EvalError;

var RangeErrorJS = RangeError;

var ReferenceErrorJS = ReferenceError;

var SyntaxErrorJS = SyntaxError;

var TypeErrorJS = TypeError;

var URIErrorJS = URIError;

var BigIntJS = BigInt;

var DateJS = Date;

var MathJS = Math;

var NumberJS = Number;

var RegExpJS = RegExp;

var StringJS = String;

var ArrayJS = Array;

var BigInt64ArrayJS = BigInt64Array;

var BigUint64ArrayJS = BigUint64Array;

var Float32ArrayJS = Float32Array;

var Float64ArrayJS = Float64Array;

var Int8ArrayJS = Int8Array;

var Int16ArrayJS = Int16Array;

var Int32ArrayJS = Int32Array;

var Uint8ArrayJS = Uint8Array;

var Uint8ClampedArrayJS = Uint8ClampedArray;

var Uint16ArrayJS = Uint16Array;

var Uint32ArrayJS = Uint32Array;

var MapJS = Map;

var SetJS = Set;

var WeakMapJS = WeakMap;

var WeakSetJS = WeakSet;

var ArrayBufferJS = ArrayBuffer;

var AtomicsJS = Atomics;

var DataViewJS = DataView;

var JSONJS = JSON;

var SharedArrayBufferJS = SharedArrayBuffer;

var Intl$1 = IntlJS;
var Promise$1 = PromiseJS;
var Proxy$1 = ProxyJS;
var Reflect$1 = ReflectJS;
var WebAssembly$1 = WebAssemblyJS;
// Value properties
var Infinity$1 = InfinityJS;
var NaN$1 = NaNJS;
var undefined$1 = undefinedJS;
// Function properties
var decodeURI$1 = decodeURIJS;
var decodeURIComponent$1 = decodeURIComponentJS;
var encodeURI$1 = encodeURIJS;
var encodeURIComponent$1 = encodeURIComponentJS;
var isFinite$1 = isFiniteJS;
var isNaN$1 = isNaNJS;
var parseFloat$1 = parseFloatJS;
var parseInt$1 = parseIntJS;
// Fundamental objects
var Boolean$1 = BooleanJS;
var Function$1 = FunctionJS;
var Object$1 = ObjectJS;
var Symbol$1 = SymbolJS;
// Fundamental Error objects
var Error$1 = ErrorJS;
var EvalError$1 = EvalErrorJS;
var RangeError$1 = RangeErrorJS;
var ReferenceError$1 = ReferenceErrorJS;
var SyntaxError$1 = SyntaxErrorJS;
var TypeError$1 = TypeErrorJS;
var URIError$1 = URIErrorJS;
// Number and dates
var BigInt$1 = BigIntJS;
var Date$1 = DateJS;
var Math$1 = MathJS;
var Number$1 = NumberJS;
// Text processing
var RegExp$1 = RegExpJS;
var String$1 = StringJS;
// Indexed collections
var Array$1 = ArrayJS;
var BigInt64Array$1 = BigInt64ArrayJS;
var BigUint64Array$1 = BigUint64ArrayJS;
var Float32Array$1 = Float32ArrayJS;
var Float64Array$1 = Float64ArrayJS;
var Int8Array$1 = Int8ArrayJS;
var Int16Array$1 = Int16ArrayJS;
var Int32Array$1 = Int32ArrayJS;
var Uint8Array$1 = Uint8ArrayJS;
var Uint8ClampedArray$1 = Uint8ClampedArrayJS;
var Uint16Array$1 = Uint16ArrayJS;
var Uint32Array$1 = Uint32ArrayJS;
// Keyed collections
var Map$1 = MapJS;
var Set$1 = SetJS;
var WeakMap$1 = WeakMapJS;
var WeakSet$1 = WeakSetJS;
// Structured data
var ArrayBuffer$1 = ArrayBufferJS;
var Atomics$1 = AtomicsJS;
var DataView$1 = DataViewJS;
var JSON$1 = JSONJS;
var SharedArrayBuffer$1 = SharedArrayBufferJS;

export { Array$1 as Array, ArrayBuffer$1 as ArrayBuffer, Atomics$1 as Atomics, BigInt$1 as BigInt, BigInt64Array$1 as BigInt64Array, BigUint64Array$1 as BigUint64Array, Boolean$1 as Boolean, DataView$1 as DataView, Date$1 as Date, Error$1 as Error, EvalError$1 as EvalError, Float32Array$1 as Float32Array, Float64Array$1 as Float64Array, Function$1 as Function, Infinity$1 as Infinity, Int16Array$1 as Int16Array, Int32Array$1 as Int32Array, Int8Array$1 as Int8Array, Intl$1 as Intl, JSON$1 as JSON, Map$1 as Map, Math$1 as Math, NaN$1 as NaN, Number$1 as Number, Object$1 as Object, Promise$1 as Promise, Proxy$1 as Proxy, RangeError$1 as RangeError, ReferenceError$1 as ReferenceError, Reflect$1 as Reflect, RegExp$1 as RegExp, Set$1 as Set, SharedArrayBuffer$1 as SharedArrayBuffer, String$1 as String, Symbol$1 as Symbol, SyntaxError$1 as SyntaxError, TypeError$1 as TypeError, URIError$1 as URIError, Uint16Array$1 as Uint16Array, Uint32Array$1 as Uint32Array, Uint8Array$1 as Uint8Array, Uint8ClampedArray$1 as Uint8ClampedArray, WeakMap$1 as WeakMap, WeakSet$1 as WeakSet, WebAssembly$1 as WebAssembly, decodeURI$1 as decodeURI, decodeURIComponent$1 as decodeURIComponent, encodeURI$1 as encodeURI, encodeURIComponent$1 as encodeURIComponent, isFinite$1 as isFinite, isNaN$1 as isNaN, parseFloat$1 as parseFloat, parseInt$1 as parseInt, undefined$1 as undefined };
