// vanillajs/dest/cjs/index.js
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

var IntlJS = Intl;

var PromiseJS = Promise;

var ProxyJS = Proxy;

var ReflectJS = Reflect;

var WebAssemblyJS = WebAssembly;

var InfinityJS = Infinity;

var NaNJS = NaN;

var undefinedJS = undefined;

var decodeURIJS = decodeURI;

var decodeURIComponentJS = decodeURIComponent;

var encodeURIJS = encodeURI;

var encodeURIComponentJS = encodeURIComponent;

// eslint-disable-next-line no-restricted-globals
var isFiniteJS = isFinite;

// eslint-disable-next-line no-restricted-globals
var isNaNJS = isNaN;

var parseFloatJS = parseFloat;

var parseIntJS = parseInt;

var BooleanJS = Boolean;

var FunctionJS = Function;

var ObjectJS = Object;

var SymbolJS = Symbol;

var ErrorJS = Error;

var EvalErrorJS = EvalError;

var RangeErrorJS = RangeError;

var ReferenceErrorJS = ReferenceError;

var SyntaxErrorJS = SyntaxError;

var TypeErrorJS = TypeError;

var URIErrorJS = URIError;

var BigIntJS = BigInt;

var DateJS = Date;

var MathJS = Math;

var NumberJS = Number;

var RegExpJS = RegExp;

var StringJS = String;

var ArrayJS = Array;

var BigInt64ArrayJS = BigInt64Array;

var BigUint64ArrayJS = BigUint64Array;

var Float32ArrayJS = Float32Array;

var Float64ArrayJS = Float64Array;

var Int8ArrayJS = Int8Array;

var Int16ArrayJS = Int16Array;

var Int32ArrayJS = Int32Array;

var Uint8ArrayJS = Uint8Array;

var Uint8ClampedArrayJS = Uint8ClampedArray;

var Uint16ArrayJS = Uint16Array;

var Uint32ArrayJS = Uint32Array;

var MapJS = Map;

var SetJS = Set;

var WeakMapJS = WeakMap;

var WeakSetJS = WeakSet;

var ArrayBufferJS = ArrayBuffer;

var AtomicsJS = Atomics;

var DataViewJS = DataView;

var JSONJS = JSON;

var SharedArrayBufferJS = SharedArrayBuffer;

var Intl$1 = IntlJS;
var Promise$1 = PromiseJS;
var Proxy$1 = ProxyJS;
var Reflect$1 = ReflectJS;
var WebAssembly$1 = WebAssemblyJS;
// Value properties
var Infinity$1 = InfinityJS;
var NaN$1 = NaNJS;
var undefined$1 = undefinedJS;
// Function properties
var decodeURI$1 = decodeURIJS;
var decodeURIComponent$1 = decodeURIComponentJS;
var encodeURI$1 = encodeURIJS;
var encodeURIComponent$1 = encodeURIComponentJS;
var isFinite$1 = isFiniteJS;
var isNaN$1 = isNaNJS;
var parseFloat$1 = parseFloatJS;
var parseInt$1 = parseIntJS;
// Fundamental objects
var Boolean$1 = BooleanJS;
var Function$1 = FunctionJS;
var Object$1 = ObjectJS;
var Symbol$1 = SymbolJS;
// Fundamental Error objects
var Error$1 = ErrorJS;
var EvalError$1 = EvalErrorJS;
var RangeError$1 = RangeErrorJS;
var ReferenceError$1 = ReferenceErrorJS;
var SyntaxError$1 = SyntaxErrorJS;
var TypeError$1 = TypeErrorJS;
var URIError$1 = URIErrorJS;
// Number and dates
var BigInt$1 = BigIntJS;
var Date$1 = DateJS;
var Math$1 = MathJS;
var Number$1 = NumberJS;
// Text processing
var RegExp$1 = RegExpJS;
var String$1 = StringJS;
// Indexed collections
var Array$1 = ArrayJS;
var BigInt64Array$1 = BigInt64ArrayJS;
var BigUint64Array$1 = BigUint64ArrayJS;
var Float32Array$1 = Float32ArrayJS;
var Float64Array$1 = Float64ArrayJS;
var Int8Array$1 = Int8ArrayJS;
var Int16Array$1 = Int16ArrayJS;
var Int32Array$1 = Int32ArrayJS;
var Uint8Array$1 = Uint8ArrayJS;
var Uint8ClampedArray$1 = Uint8ClampedArrayJS;
var Uint16Array$1 = Uint16ArrayJS;
var Uint32Array$1 = Uint32ArrayJS;
// Keyed collections
var Map$1 = MapJS;
var Set$1 = SetJS;
var WeakMap$1 = WeakMapJS;
var WeakSet$1 = WeakSetJS;
// Structured data
var ArrayBuffer$1 = ArrayBufferJS;
var Atomics$1 = AtomicsJS;
var DataView$1 = DataViewJS;
var JSON$1 = JSONJS;
var SharedArrayBuffer$1 = SharedArrayBufferJS;

exports.Array = Array$1;
exports.ArrayBuffer = ArrayBuffer$1;
exports.Atomics = Atomics$1;
exports.BigInt = BigInt$1;
exports.BigInt64Array = BigInt64Array$1;
exports.BigUint64Array = BigUint64Array$1;
exports.Boolean = Boolean$1;
exports.DataView = DataView$1;
exports.Date = Date$1;
exports.Error = Error$1;
exports.EvalError = EvalError$1;
exports.Float32Array = Float32Array$1;
exports.Float64Array = Float64Array$1;
exports.Function = Function$1;
exports.Infinity = Infinity$1;
exports.Int16Array = Int16Array$1;
exports.Int32Array = Int32Array$1;
exports.Int8Array = Int8Array$1;
exports.Intl = Intl$1;
exports.JSON = JSON$1;
exports.Map = Map$1;
exports.Math = Math$1;
exports.NaN = NaN$1;
exports.Number = Number$1;
exports.Object = Object$1;
exports.Promise = Promise$1;
exports.Proxy = Proxy$1;
exports.RangeError = RangeError$1;
exports.ReferenceError = ReferenceError$1;
exports.Reflect = Reflect$1;
exports.RegExp = RegExp$1;
exports.Set = Set$1;
exports.SharedArrayBuffer = SharedArrayBuffer$1;
exports.String = String$1;
exports.Symbol = Symbol$1;
exports.SyntaxError = SyntaxError$1;
exports.TypeError = TypeError$1;
exports.URIError = URIError$1;
exports.Uint16Array = Uint16Array$1;
exports.Uint32Array = Uint32Array$1;
exports.Uint8Array = Uint8Array$1;
exports.Uint8ClampedArray = Uint8ClampedArray$1;
exports.WeakMap = WeakMap$1;
exports.WeakSet = WeakSet$1;
exports.WebAssembly = WebAssembly$1;
exports.decodeURI = decodeURI$1;
exports.decodeURIComponent = decodeURIComponent$1;
exports.encodeURI = encodeURI$1;
exports.encodeURIComponent = encodeURIComponent$1;
exports.isFinite = isFinite$1;
exports.isNaN = isNaN$1;
exports.parseFloat = parseFloat$1;
exports.parseInt = parseInt$1;
exports.undefined = undefined$1;
